03power.cpp

```diff
@@ -0,0 +1,68 @@
+#include <iostream>
+
+using namespace std;
+
+double numberInPower(double numberX, int powerN);
+void numberXInput(double* numberX);
+void powerNInput(int* powerN);
+
+int main()
+{
+	double numberX, numberXInPowerN;
+	int powerN;
+
+	cout << "This program raises the number x to the power n." << endl << endl;
+
+	numberXInput(&numberX);
+	powerNInput(&powerN);
+
+	if (numberX == 0)
+	{
+		cout << endl << "Error! Invalid number x = 0." << endl << endl;
+
+		system("pause");
+		return -1;
+	}
+
+	numberXInPowerN = numberInPower(numberX, powerN);
+
+	cout << endl << "The number x = " << numberX << " in the power of n = " << powerN << " is " << numberXInPowerN << endl << endl;
+
+	system("pause");
+	return 0;
+}
+
+void powerNInput(int* powerN)
+{
+	cout << "Enter your power n: ";
+	cin >> *powerN;
+}
+
+void numberXInput(double* numberX)
+{
+	cout << "Enter your number x: ";
+	cin >> *numberX;
+}

```

Comment:
the same functions

Reply:
I don't know how to make one function instead of two as they work with int and double. Could you please help me?

Reply:
I made a function as below
![image](https://user-images.githubusercontent.com/79978383/111083584-561aa800-851f-11eb-929c-5ae21f8c4b6f.png)
and used the next code in main():
![image](https://user-images.githubusercontent.com/79978383/111083634-895d3700-851f-11eb-98dc-0ff3d7834773.png)

As for me, it is the easiest way in terms of functions. But it is easier to use `cin >>` instead of function